spec/requests/todos_spec.rb

```diff
@@ -12,7 +12,12 @@
     end
 
     describe 'GET #index' do
-      it 'TODOの一覧を取得できること' do
+      let!(:todo) { create(:todo) }
+      context 'TODOが作成されているとき' do
+        it 'TODOの一覧を取得できること' do
+          get "/api/todos"
+          expect(response).to have_http_status(200)
+        end
```

Comment:
itを追加して,
request.bodyの中身を確認する形にした方がいいかもですね
```suggestion
　it 'responseの中身が正しいこと'
        expect(response.body.count).to eq 1 # 帰ってきたtodoの配列数が1を確認
        expect(response.body[0].id).to eq todo.id
        expect(response.body[0].text).to eq todo.text
    end
```

ローカルで動かしてないので少し微調整が必要かもですが、こんな感じで中身を確認するテストを追加してもいいかもです！

Reply:
itを追加して、元のitも少し命名を変更しました。
```
it 'TODOの一覧を取得できること' do # 修正前 
it 'responseのステータスが200であること' do # 修正後

it 'responseの中身が正しいこと' do # 追加
```